src/scales/scale.linear.js

```diff
@@ -156,7 +158,7 @@ module.exports = function(Chart) {
 			}
 		},
 		getLabelForIndex: function(index, datasetIndex) {
```

Comment:
this method is the same in both scales. perhaps we should move it to `core.scale.js` or `scale.linearbase`. that seems like a better solution than introducing a new method `getScaleLabel` since we generally try to avoid introducing new public APIs except when required

Reply:
done